### PR TITLE
Fix nativeMethods.performanceNow in IE9 (closes #670)

### LINKED ISSUE #670: nativeMethods.performanceNow throw an error in IE9

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by miherlosev** [comment]
lgtm

File: test/client/fixtures/sandbox/native-methods-test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+test('performanceNow', function () {
+    if (!nativeMethods.performanceNow) {
+        expect(0);
+        return;
+    }
+
+    var now = nativeMethods.performanceNow();
+
+    ok(!isNaN(parseFloat(now)));
+});
```
Comment (churkin) [inline test/client/fixtures/sandbox/native-methods-test.js]:
```js
if (nativeMethods.performanceNow) {
    test('performanceNow', function () {
         var now = nativeMethods.performanceNow();
 
         ok(!isNaN(parseFloat(now)));
     });
}
```
??

**Comment by AlexanderMoskovkin** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Pull request merged**